html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="#" method="#" id="coding-form">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+
+      <div>
+        <label for="why">Why is coding cool?</label>
+      </div>
+      <select name="why">
+        <option value="">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <label for="language"><p>Favorite Language (choose one)</p></label>
+      </div>
+      <div>
+        <input type="radio" name="language" value="HTML"> HTML
```

Comment:
you will need a label for each input, its common to wrap them around radio inputs like this: 
```html
      <label>
        <input type="radio" name="favoriteLanguage" value="JavaScript">
        JavaScript
      </label>
```

Reply:
For the radio buttons, do the labels need a for attribute as well, or just <label>?
![Screenshot 2022-01-27 201548](https://user-images.githubusercontent.com/97073517/151486376-081cdef5-3cab-4dd4-b760-1a0955faf519.png)

Reply:
![image](https://user-images.githubusercontent.com/97073517/151487302-a3d11c57-edfb-4fc4-a35d-e0dadeacf90e.png)